multicluster/cmd/multicluster-controller/member.go

```diff
@@ -122,15 +121,16 @@ func runMember(o *Options) error {
 		return fmt.Errorf("error creating Node controller: %v", err)
 	}
 
-	staleController := multiclustercontrollers.NewStaleResCleanupController(
+	staleController := member.NewMemberStaleResCleanupController(
 		mgr.GetClient(),
 		mgr.GetScheme(),
+		clusterSetReconciler.CommonAreaCreated,
```

Comment:
Probably better to initialize the channel in this func and pass it to both `NewMemberClusterSetReconciler `and `NewMemberStaleResCleanupController`.

Reply:
Renamed to CommonAreaCreationCh, and initialized it in member.go.